### PR TITLE
Updated prefwindow.dtd

### PR DESCRIPTION
**Pull request by mauricepape**
Merged changes from the BabelZilla WTS

### CODE REVIEW

File: locale/de/prefwindow.dtd [hunk 1]
```diff
@@ -4,12 +4,12 @@
 <!ENTITY advanced.label "Erweitert">
 <!ENTITY about.label "Über">
 <!ENTITY copyright.label "Copyright © 2012,2013,2015 Jonathan Kamens">
-<!ENTITY license.label "Released under the Mozilla Public License, version 2.0">
-<!ENTITY support.label "Bitte unterstützen Sie die Entwicklung dieses und anderen tollen Add-Ons!">
+<!ENTITY license.label "Veröffentlicht unter der Mozilla Public License, version 1.1">
```
Comment (jikamens) [inline locale/de/prefwindow.dtd]: Please change the 1.1 back to 2.0. Thanks.

Comment (mauricepape) [inline locale/de/prefwindow.dtd]: Done, see 2c9b7a7 below. You might want to change the number on the BabelZilla WTS page as well.

Comment (jikamens) [inline locale/de/prefwindow.dtd]: Sorry to be a stick-in-the-mud, but can you squash the two commits into one before I merge?

Unfortunately, BabelZilla's license drop-down doesn't offer MPL 2.0 as an option. :-(

Comment (mauricepape) [inline locale/de/prefwindow.dtd]: I'm not really a professional with git, but I think the squash should be done now.

Sounds like a suggestion for to me :P

**Pull request merged**